README.md

```diff
@@ -23,6 +23,8 @@ If you installed `keras-retinanet` correctly, the train script will be installed
 However, if you make local modifications to the `keras-retinanet` repository, you should run the script directly from the repository.
 That will ensure that your local changes will be used by the train script.
 
+The default backbone is 'resnet50'. You can change this using the '--backbone=xxx' argument in the running script.
```

Comment:
Ah apologies, I actually only meant the part saying "More backbones will be added in the future". The list of accepted backbones is useful here I'd say.

Reply:
Ok!